spec/spec_helper.rb

```diff
@@ -14,23 +14,26 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 require 'securerandom'
-require 'simplecov'
-require 'simplecov-cobertura'
-SimpleCov.formatters = [
-  SimpleCov::Formatter::HTMLFormatter,
-  SimpleCov::Formatter::CoberturaFormatter,
-]
-
-# Give each coverage object a unique ID based on the matrix name and number from parallel tests
-command_name = 'RSpec'
-command_name += "-#{ENV['TEST_MATRIX_NODE_NAME']}"      if ENV['TEST_MATRIX_NODE_NAME']
-command_name += "-#{ENV['DEFAULT_FEATURE_FLAG_STATE']}" if ENV['DEFAULT_FEATURE_FLAG_STATE']
-command_name += "-#{ENV['TEST_ENV_NUMBER']}"            if ENV['TEST_ENV_NUMBER']
-
-SimpleCov.command_name(command_name)
-
-SimpleCov.start 'rails' do
-  enable_coverage :branch
+
+if ENV.fetch('COVERAGE', 'false') == 'true'
```

Comment:
perhaps `if ENV.fetch('COVERAGE', 'true') == 'true'` is safer here, then in local development we can just set `COVERAGE` to false. whaddayathink?

Reply:
Sure.  Is there somewhere we can put it where it'll be turned off for new developers?

Reply:
Does that run in `test`? (which is where the coverage is running)

I guess https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/.env.test instead?

Reply:
cool, `.env.test` is a good shout